src/styles/mixins/_typography.scss

```diff
@@ -42,3 +42,30 @@
     line-height: $bdl-line-height;
     letter-spacing: .3px;
 }
+
+@mixin target-ie11 {
+    @media all and (-ms-high-contrast: none), (-ms-high-contrast: active) {
+        @content;
+    }
+}
+
+@mixin line-clamp($max-lines: 2, $line-height: 1.5) {
```

Comment:
New mixins seem to use a `bdl` prefix so I think we should add that

Reply:
We are already using this `line-clamp` mixin in EUA.

My hope was I could...

- introduce it in BUIE since we have some strings that need it
- import it back to EUA and update to pull it from BUIE
- remove it from EUA to keep a single source of truth.

My understanding is people have been using the mixin as `line-clamp` already.  So renaming might complicate things.

Reply:
Gotcha. I think it would be good to give it a namespace (same with `target-ie11`) so there's no naming conflicts when used in parent apps and it's more apparent/explicit that this mixin is coming from BUIE.